src/sidebar/util/fetch-config.js

```diff
@@ -57,6 +57,7 @@ function fetchConfigFromAncestorFrame(origin, window_ = window) {
       ancestor,
       origin,
       'requestConfig',
+      [],
```

Comment:
This missing parameter was an actual mistake caught by typechecking. The tests for this module made the same mistake, so they didn't catch the problem.